.changeset/shaggy-schools-sing.md

```diff
@@ -0,0 +1,15 @@
+---
+"@spectrum-css/search": major
+"@spectrum-css/textfield": patch
```

Comment:
This may not have to be textfield now, and should be tokens? Since we added the custom corner-radius-1000?